packages/rest/src/rest.server.ts

```diff
@@ -297,10 +297,19 @@ export class RestServer extends Context implements Server, HttpServerLike {
     return handleUnExpectedError;
   }
 
+  /**
+   * Non-null assertion type guide
+   * @param val - A value can be `undefined` or `null`
+   */
+  private assert<T>(val: T | null | undefined): asserts val is T {
```

Comment:
Is there any particular reason for making this function a class member? AFAICT, it does not need anything from the RestServer class. Also the name `assert` is not very intent-revealing, have you considered something like `assertExists`?

```ts
function assertExists<T>(val: T | null | undefined): asserts val is T {
  assert(val);
}
```